File: algorithmic_toolbox/week_3/02_greedy_algorithms_starter_files/change/Change.java
import java.util.Scanner;

public class Change {
	private static int getChange(int m) {
		int[] coins = { 10, 5, 1 };
		int count = 0;

		for (int i = 0; m > 0; i++) {
			count += m / coins[i];
			m %= coins[i];
		}
		return count;
	}

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int m = scanner.nextInt();
		System.out.println(getChange(m));

	}
}

File: algorithmic_toolbox/week_3/02_greedy_algorithms_starter_files/change/change.cpp
#include <iostream>

int get_change(int m) {
    int coins[] = {10, 5, 1};
    int count = 0;
    for (int i = 0; m > 0; i++) {
        count += m / coins[i];
        m %= coins[i];
    }
    return count;
}

int main() {
    int m;
    std::cin >> m;
    std::cout << get_change(m) << '\n';
}
